src/main/java/com/solvd/rentalcompany/entities/GasVehicles.java

```diff
@@ -0,0 +1,35 @@
+package com.solvd.rentalcompany.entities;
+
+import com.solvd.rentalcompany.abstractClasses.Vehicle;
+
+public class GasVehicles {
+
+     public Vehicle vehicle;
```

Comment:
GasVehicle should extend Vehicle
We can use Engine, Wheels... as inner classes, in this case, we should extend from Vehicle